todo/src/components/Todo.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { toggleTodo } from './actions';
+
+const Todo = props => {
+  console.log(props)
+return (
+  
+  <h2 className={props.todo.completed ? 'completed' : 'not yet'} onClick={( ) => props.toggleTodo(props.todo.todo)}>{props.todo.todo}</h2>
+)
```

Comment:
While you can change your todos from completed to not, and back again, you can't really tell on the front end. One easy way to implement this would have been: ` <h2 style={props.todo.completed ? {textDecoration: 'line-through'} : null}`